onnx/shape_inference/shape_inference.h

```diff
@@ -34,15 +34,15 @@ struct InferenceContextImpl : public InferenceContext {
 void InferShapes(ModelProto& m) {
   auto* g = m.mutable_graph();
 
-  std::unordered_map<std::string, TypeProto_Tensor> valueTypesByName;
```

Comment:
can we rename this file please? I'm seeing two files with name "shape_inference.h".